From ee73ea5ddd1c5a7bfdefe03fb006cd46a260fb69 Mon Sep 17 00:00:00 2001
From: Arun Ravindran <arun.ravindran@intel.com>
Date: Tue, 16 Apr 2013 12:50:07 +0300
Subject: [PDK MR2 ABSP] redhookbay: Change apns-conf.xml version

BZ: 100903

PDK uses version 8 and the version difference creates crash
in TelephonyProvider, thus preventing PDP activation.

Change-Id: Iadc0224fce96204623a00c2af03b2d420525eefe
Signed-off-by: Arun Ravindran <arun.ravindran@intel.com>
---
 apns-conf.xml |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/apns-conf.xml b/apns-conf.xml
index d5200ad..076609b 100644
--- a/apns-conf.xml
+++ b/apns-conf.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"
 standalone="no"?>
-<apns version="7">
+<apns version="8">
   <apn carrier="2 degrees mobile"
        apn="2degrees"
        proxy="118.148.1.118"
-- 
1.7.4.1

